gui/additionsPane.py

```diff
@@ -57,6 +58,7 @@ def __init__(self, parent):
         projectedImg = BitmapLoader.getImage("projected_small", "gui")
         gangImg = BitmapLoader.getImage("fleet_fc_small", "gui")
         cargoImg = BitmapLoader.getImage("cargo_small", "gui")
+        notesImg = BitmapLoader.getImage("skill_small", "gui")
```

Comment:
You wouldn't :)

This is fine, I didn't notice which icon you'd used and didn't check the other tabs to see the size.